buildSrc/build.gradle

```diff
@@ -190,6 +190,15 @@ if (project != rootProject) {
 
   // This can't be an RandomizedTestingTask because we can't yet reference it
   task integTest(type: Test) {
+    // We don't publish plugins to maven so these won't be in the testing repo.
+    // Make one random plugin available there for testing purposes.
+    dependsOn project(':plugins:ingest-geoip').getTasks().matching { it.name == "bundlePlugin" }
```

Comment:
I don't think we should be making these cross project dependencies. They cause weird ordering of tasks that become difficult to debug (eg look at the current integ test here which builds and tests the example plugins, which is impossible to debug because it lacks the output of the underlying tasks). Can we instead have a dummy project inside buildSrc used for testing of this?

Reply:
Could we have a dummy plugin under testkit? The dependency can be on the project itself (the plugin zip is the default artifact produced iirc).

My concern, as it was before (and I think has been proven right by many recent cryptic build failures) is triggering other projects to build when running tests in buildSrc. I would like to see all buildSrc tests independent of the rest of the build.

Reply:
I did that and consolidated everything to a single testkit project that has a dummy plugin to remove external reliance.